### DISCUSSION TITLE
Network Access from addon

### DISCUSSION THREAD

**Discussion by weissjuergen** [OpenCCU/OpenCCU#1991] [🇺🇸 Q&A]
I have installed the addon on HAOS (x86). Regular functionality has been tested ok. When it comes to services that require dns/routing to either my internal network or to the internet, it seems there are some limitations which prevent me to hold on my migration from RASPI hardware to this nice addon. Are there any options that allow me transparent access including internal and external dns name resolution ? 
In the pas I always avoided using hard coded  IP adresses and worked with names.

**Comment by jens-maus** [comment]
Sorry, what exactly are you talking about? Provide examples and more detailed explainations of what exactly is not working how you would expect it.

**Reply by weissjuergen** [reply]
As an example I accessed my tasmota WIFI devices through CUXD commands using their DNS names as a reference (Example Name S20-02). They are not found while using the addon. They seem to react when using a hard coded IP address instead or once I add the domain suffix in my case S20-02.fritz.box. I was hoping to avoid touching every single tasmota device to reconfigure (having dozents).

**Reply by jens-maus** [reply]
This is absolutely no limitation of RaspberryMatic or the HA addon it provides. This is a pure DNS issue you are having. Sorry, but here is the wrong place to discuss something like this. Go to a HomeAssistant discussion fora to discuss solutions for it.

**Reply by weissjuergen** [reply]
Thank you, will do.
